windows/win_iis_webapppooldefaults.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Adam Bertram (@adbertram)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = '''
+---
+module: win_iis_webapppooldefaults
+version_added: "2.2"
+short_description: Sets IIS settings for the application pool defaults
+description:
+     - Sets IIS settings for the application pool defaults
+options:
+  idleTimeout:
+    description:
```

Comment:
We try and use "pythonic" parameters. For example "idle_timeout" instead of "idleTimeout" and such.